### DISCUSSION TITLE
Cabinets won't load

### DISCUSSION THREAD

**Discussion by K3NT1N34** [opendcim/openDCIM#1573] [Q&A]
Hello Guys,
First of all, many thanks for your awesome job!

FYI - I have very basic linux knowledge.
That being said I had setup openDCIM on a debian 12.
I had import data from an existing DCIM, files and DB.

Everything is working fine instead of cabinets graphical view :
![image](https://github.com/user-attachments/assets/b21e9c7e-bba0-4db3-94e8-ccc6572f593b)

When I inspect the page I found an error 500 :
GET http://192.168.21.4/api/v1/cabinet/37/getpictures 500 (Internal Server Error) jquery.min.js:2 

Here is what I get from apache logs :
![image](https://github.com/user-attachments/assets/a022077e-1d75-418d-8311-ac7ff16295ce)

And here is the /api/v1/.htaccess file content :
![image](https://github.com/user-attachments/assets/9132a1ff-9cf3-4109-8c28-399bb1de8940)

I don't know what this file is supposed to do or either the RewritEngine.
I tried to comment out the first line then the error did point to RewriteBase instead of RewriteEngine.
I tried to comment everything then I had a 404 error.

Unfortunately I did not found the answer on existing discussions.
I hope that I have provided enough information for you to understand what's is happening.
Thanks in advance for your needful help.

**Comment by K3NT1N34** [comment]
I finally found the answer on stackoverflow which was really basic in fact.
If someone else is having the same issue the solution is below.
Just enable rewrite module and restart apache.
#sudo a2enmod rewrite
#systemctl restart apache2.service
Thanks again.

**Comment by wilpig** [comment]
Do I need to add a straight Debian install to the wiki too?

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by K3NT1N34** [reply]
I don't know, both Ubuntu and Debian and pretty similar in term of install. A basic copy/paste should work.
In fact I didn't go through enabling this module because it was part of SSL and I don't need it, at the end I just had to read more carefully the install guide :)
